app/section/sections/mobile-first/ui.test.js

```diff
@@ -0,0 +1,175 @@
+import nock from 'nock';
+import { Selector, ClientFunction } from 'testcafe';
+import { ManifestProvider } from '../../../manifestProvider';
+import dashboardWithCompleteSections from '../../../../fixtures/dashboardWithCompleteSections.json';
+
+const mobileFirstMarketingData = {
+  'mobile-first-design': 'Yes',
+};
+
+const mocks = (withMarketingData) => {
+  if (withMarketingData) {
+    nock('http://localhost:8080')
+      .get('/api/v1/Solutions/S100000-001/sections/mobile-first')
+      .reply(200, mobileFirstMarketingData);
+  } else {
+    nock('http://localhost:8080')
+      .get('/api/v1/Solutions/S100000-001/sections/mobile-first')
+      .reply(200, {});
+  }
+};
+
+const pageSetup = async (t, withMarketingData = false) => {
+  mocks(withMarketingData);
+  await t.navigateTo('http://localhost:1234/S100000-001/section/mobile-first');
+};
+
+fixture.only('Show Plug-ins Or Extensions page')
+  .afterEach(async (t) => {
+    const isDone = nock.isDone();
+    if (!isDone) {
+      nock.cleanAll();
+    }
+
+    await t.expect(isDone).ok('Not all nock interceptors were used!');
+  });
+
+test('should render the Mobile First page title', async (t) => {
+  await pageSetup(t);
+
+  const title = Selector('[data-test-id="section-title"]');
+
+  await t
+    .expect(title.innerText).eql('Native mobile or tablet - mobile first');
+});
+
+test('should render main advice of section', async (t) => {
+  await pageSetup(t);
+
+  const mainAdvice = Selector('[data-test-id="section-main-advice"]');
+
+  await t
+    .expect(mainAdvice.innerText).eql('Answer the following questions about your native mobile or tablet Solution');
+});
+
+test('should render all the advice of the section', async (t) => {
+  await pageSetup(t);
+
+  const sectionManifest = new ManifestProvider().getSectionManifest('mobile-first');
+  const expectedAdditionalAdvice = sectionManifest.additionalAdvice.join('\n\n');
+
+  const additionalAdvice = Selector('[data-test-id="section-additional-advice"]');
+
+  await t
+    .expect(additionalAdvice.innerText).eql(expectedAdditionalAdvice);
+});
+
+test('should render the mobile first design question', async (t) => {
+  await pageSetup(t);
+
+  const pluginsRequiredQuestion = Selector('[data-test-id="question-mobile-first-design"]');
+
+  await t
+    .expect(pluginsRequiredQuestion.find('.nhsuk-fieldset__legend').innerText).eql('Was this Solution design with a mobile first approach?')
```

Comment:
Careful with variable names. The variable 'pluginsRequiredQuestion' could do with a better name